### ISSUE TITLE
Seahub docker container not starting properly after the first time.

### ISSUE THREAD

**Issue by caleb85** [ggogel/seafile-containerized#4]
When starting up the seahub container after staring it the first time it fails to load.  Once you log into the seafile web interface it gives a page unavailable error. This only happens after the first time you restart the container.  I've already tried giving the seahub container a longer time to restart / shut down, but it still happens. Manually deleting the seafile.sock file causes it to work without restarting the container. 
 Error log: 
```
2021/02/19 02:36:02 socat[11664] N exit(1)
Starting socat...
2021/02/19 02:36:02 socat[11666] E "/opt/seafile/seafile-server-latest/runtime/seafile.sock" exists
2021/02/19 02:36:02 socat[11666] N exit(1)
Starting socat...
2021/02/19 02:36:02 socat[11668] E "/opt/seafile/seafile-server-latest/runtime/seafile.sock" exists
2021/02/19 02:36:02 socat[11668] N exit(1)
```

compose file
```
version: '3.8'
services:
  seafile-server:
    image: ggogel/seafile-server:8.0.3
    volumes:
      - seafile-data:/shared
    environment:
      - DB_HOST=db
      - DB_ROOT_PASSWD=hunter2
      - TIME_ZONE=America/Detroit
      - HTTPS=false # Set this to true if you plan to use a reverse proxy with HTTPS. Can be changed later in the admin settings on the web-ui.
      - SEAFILE_SERVER_HOSTNAME=203.0.113.2 # Mandatory on first deployment!
    depends_on:
      - db
      - memcached
      - seafile-caddy
    networks:
      - seafile-net

  seahub:
    image: ggogel/seahub:8.0.3
    volumes:
      - seafile-data:/shared
      - seahub-avatars:/shared/seafile/seahub-data/avatars
      - seahub-custom:/shared/seafile/seahub-data/custom
    environment:
      - SEAFILE_ADMIN_EMAIL=me@email.com
      - SEAFILE_ADMIN_PASSWORD=asecret
    depends_on:
      - seafile-server
      - seafile-caddy
    networks:
      - seafile-net

  seahub-media:
    image: ggogel/seahub-media:8.0.3
    volumes:
     - seahub-avatars:/usr/share/caddy/media/avatars
     - seahub-custom:/usr/share/caddy/media/custom
    depends_on:
      - seafile-server
      - db
      - memcached
      - seafile-caddy
    networks:
      - seafile-net

  db:
    image: mariadb:10.5.8
    environment:
      - MYSQL_ROOT_PASSWORD=hunter2
      - MYSQL_LOG_CONSOLE=true
    volumes:
      - seafile-mariadb:/var/lib/mysql
    networks:
      - seafile-net

  memcached:
    image: memcached:1.6.9
    entrypoint: memcached -m 1024
    networks:
      - seafile-net

  seafile-caddy:
    image: ggogel/seafile-caddy:1.0.0
    ports:
      - "80:80" # Point your reverse proxy to port 80 of this service
    networks:
      - seafile-net
      - default

networks:
  seafile-net:
    internal: true

volumes:
  seafile-data:
  seafile-mariadb:
  seahub-avatars:
  seahub-custom:
```
P.s Thanks for creating the docker containers. This was by far the easiest way to set up seafile.

**Comment by ggogel** [comment]
Hi, thanks for the issue report. This is indeed an edge case that I had not covered yet. I'm personally running Docker Swarm and there you usually don't restart containers but instead just stop the old one and it'll automatically start a new one. 

To fix this, I've added the unlink-early option to socat: 4bd4b8f. This option deletes the socket file if it exists. This is now on `ggogel/seahub:unstable`. Can you test if this resolves the issue for you?

**Comment by caleb85** [comment]
Added the unlink-early option to socat fix the issue.

**Comment by ggogel** [comment]
Thanks. I've now released the fix in `seahub-8.0.3_006`.
